src/main/java/gg/solarmc/solarcredits/CreditCommands.java

```diff
@@ -0,0 +1,242 @@
+package gg.solarmc.solarcredits;
+
+import com.google.gson.JsonParser;
+import gg.solarmc.loader.credits.CreditsKey;
+import gg.solarmc.loader.credits.WithdrawResult;
+import gg.solarmc.solarcredits.menus.ConfirmMenu;
+import okhttp3.FormBody;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.function.Consumer;
+
+public class CreditCommands implements CommandExecutor {
+
+    private final SolarCredit plugin;
+    private static final Logger logger = LoggerFactory.getLogger(CreditCommands.class);
+
+    private final String X_TEBEX_SECRET;
+
+    public CreditCommands(SolarCredit plugin, @NotNull String TebexSecret) {
```

Comment:
```suggestion
    public CreditCommands(SolarCredit plugin, String TebexSecret) {
```
I recommend using a nonnull-by-default policy. Most tools have the possibility to configure this. It's the way Aesthetik and I develop - never null unless documented otherwise.